src/Pages/ContentListTable/ContentListTable.tsx

```diff
@@ -167,6 +169,15 @@ const ContentListTable = () => {
   const introspectRepoForUuid = (uuid: string): Promise<void> =>
     introspectRepository({ uuid: uuid, reset_count: true } as IntrospectRepositoryRequestItem);
 
+  const { mutateAsync: triggerSnapshotMutation } = useTriggerSnapshot(queryClient);
+  triggerSnapshotMutation;
+  const { mutateAsync: introspectSnapshotMutation } = useTriggerSnapshot(queryClient);
+  introspectSnapshotMutation;
+
+  const triggerSnapshot = async (uuid: string): Promise<void> => {
```

Comment:
This is being called below on line 295, it is what is called when you "click" your kebab action called "Trigger Snapshot" 

So if you say, wanted to call an API.. it might be a good idea to do it here. 😉